CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 Change Log / Release Log for mallocMC
 ================================================================
 
+2.3.1crp
+--------
+**Date:** 2019-02-14
+
+A critical bug was fixed which can result in an illegal memory access.
+
+### Changes to mallocMC 2.3.0crp
+
+**Bug fixes**
+ - fix illegal memory access in `XMallocSIMD` #150
+
```

Comment:
Let us add #154 while we are on it :)

Reply:
Yes, not good if we work not side by side.^^ I will add it to the changelog and do a break to avoid double work^^